### DISCUSSION TITLE
I think there should be more diagrams in the documentation

### DISCUSSION THREAD

**Discussion by kochrt** [vikejs/vike#1260] [Discussion]
It would help me, anyway, to have some diagrams to reference when trying to think about what `pageContext` is at a certain time, when `onBeforeRender` is called, where the boundary between the server and the client is; stuff like that. 

Something like https://v2.nuxt.com/docs/concepts/nuxt-lifecycle/

**Comment by brillout** [comment]
👍 I agree and we plan to add some once the `v1.0.0` is released.
